apps/backoffice-v2/src/common/hooks/useZodSearchParams/dateRangeSchema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+export const dateRangeSchema = z.object({
```

Comment:
1. Notice the naming convention for schemas across the project
2. The goal for this schema is to declare the search query params for the home route, it may have more beyond from and to in the future

```suggestion
export const HomeSearchSchema = z.object({
```